aws/SDK_README.md.hb

```diff
@@ -0,0 +1,109 @@
+{{!--
+This is the README Handlebars template for `aws-sdk-rust`.
+It gets instantiated and copied into the build artifacts by the `aws:sdk:assemble` build target.
+
+Available template arguments:
+- `{{sdk_version}}`: the SDK version number (just the number, no `v` prefix)
+- `{{msrv}}`: The MSRV Rust compiler version (just the number, no `v` prefix)
+--}}
+<!--
+IMPORTANT:
+This README file is auto-generated by the build system in awslabs/smithy-rs.
+To update it, edit the `aws/SDK_README.md.hb` Handlebars template in that repository.
+-->
+
+# The AWS SDK for Rust [![Docs](https://img.shields.io/badge/docs-v{{sdk_version}}-blue)](https://awslabs.github.io/aws-sdk-rust/) ![MSRV](https://img.shields.io/badge/msrv-{{msrv}}-red) [![Usage Guide](https://img.shields.io/badge/Developer_Guide-blue)](https://docs.aws.amazon.com/sdk-for-rust/latest/dg/welcome.html)
+
+This repo contains the new AWS SDK for Rust (the SDK) and its [public roadmap](https://github.com/awslabs/aws-sdk-rust/projects/1).
+
+**Please Note: The SDK is currently released as a developer preview and is intended strictly for feedback purposes only. Do not use this SDK for production workloads.**
+
+The SDK is code generated from [Smithy models](https://awslabs.github.io/smithy/) that represent each AWS service. The code used to generate the SDK can be found in [smithy-rs](https://github.com/awslabs/smithy-rs).
+
+## Getting Started with the SDK
+
+> Examples are availble for many services and operations, check out the [examples folder](https://github.com/awslabs/aws-sdk-rust/tree/main/examples).
+
+> For a step-by-step guide including several advanced use cases, check out the [Developer Guide](https://docs.aws.amazon.com/sdk-for-rust/latest/dg/welcome.html).
+
+The SDK provides one crate per AWS service. You must add [Tokio](https://crates.io/crates/tokio) as a dependency within your Rust project to execute asynchronous code.
+
+1. Create a new Rust project: `cargo new sdk-example`
+2. Add dependencies to DynamoDB and Tokio to your **Cargo.toml** file:
+
+    ```toml
+    [dependencies]
+    aws-config = "{{sdk_version}}"
+    aws-sdk-dynamodb = "{{sdk_version}}"
+    tokio = { version = "1", features = ["full"] }
+    ```
+
+3. Provide your AWS credentials with the default credential provider chain, which currently looks in:
+   - Environment variables: `AWS_ACCESS_KEY_ID`, `AWS_SECRET_ACCESS_KEY`, and `AWS_REGION`
+   - Web Identity Token credentials from the environment or container (including EKS)
+   - The default credentials files located in `~/.aws/config` and `~/.aws/credentials` (location can vary per platform)
+   - ECS Container Credentials (IAM roles for tasks)
+   - EC2 Instance Metadata Service (IAM Roles attached to instance)
+**Note:** SSO is not supported yet.
```

Comment:
We support this now right?

Reply:
Good catch! Looks like this needs to be updated.